src/stats/src/VectorRV.C

```diff
@@ -28,6 +28,7 @@
 
 #include <queso/Defines.h>
 #include <gsl/gsl_sf_psi.h> // todo: take specificity of gsl_, i.e., make it general (gsl or boost or etc)
+#include <queso/InfoTheory.h>
```

Comment:
This is likely *adding* dependencies, not removing them as the git comment states.  Anything that included VectorRV.h but not any of its subclasses (imagine user code with a container of pointers to the base class) now has a dependency on InfoTheory.h that it didn't have before.